setup.py

```diff
@@ -137,6 +144,6 @@
     sys.exit(0)
 else:
     # build the client with an icon
-    os.system(python_path + "/Scripts/pyinstaller\" client.py " + strUPX + " --exclude-module FixTk --exclude-module tcl --exclude-module tk "
```

Comment:
From my experience it does not always get the path for every user.

Reply:
Oh ok. I changed it because my pyinstaller is located in app data, not the scripts folder. I'll change it so it checks where PyInstalller is located first, then execute the command.

Reply:
ok, thats good to know. i always thought it was in the python folder.